lib/src/conflicts.rs

```diff
@@ -77,51 +78,45 @@ fn write_diff_hunks(hunks: &[DiffHunk], file: &mut dyn Write) -> std::io::Result
     Ok(())
 }
 
-async fn get_file_contents(store: &Store, path: &RepoPath, term: &Option<FileId>) -> ContentHunk {
+async fn get_file_contents(
+    store: &Store,
+    path: &RepoPath,
+    term: &Option<FileId>,
+) -> BackendResult<ContentHunk> {
     match term {
         Some(id) => {
             let mut content = vec![];
             store
                 .read_file_async(path, id)
-                .await
-                .unwrap()
+                .await?
                 .read_to_end(&mut content)
-                .unwrap();
-            ContentHunk(content)
+                .map_err(|err| BackendError::ReadObject {
```

Comment:
maybe you mean `ReadFile`?

(alternatively, we can add a separate patch that migrates all `ReadObject` of file type to `ReadFile.)

Reply:
Ah, yes. Fixed.